connmgr/seed.go

```diff
@@ -92,7 +93,7 @@ type node struct {
 // The available filters can be set via the exported functions that start with
 // the prefix SeedFilter.  See the documentation for each function for more
 // details.
-func SeedAddrs(ctx context.Context, seeder string, dialFn DialFunc, filters ...func(f *HttpsSeederFilters)) ([]*wire.NetAddress, error) {
+func SeedAddrs(ctx context.Context, seeder string, dialFn DialFunc, filters ...func(f *HttpsSeederFilters)) ([]*addrmgr.NetAddress, error) {
```

Comment:
Breaking change to the API.  `connmgr` needs a major module version bump for this since it hasn't been done since the last release.

Reply:
Also, for reasons mentioned elsewhere, please prefer `[]addrmgr.NetAddress` over the pointer variant.